Runtime/Cesium3DTileset.cs

```diff
@@ -589,6 +589,34 @@ public bool generateSmoothNormals
             }
         }
 
+        [SerializeField]
+        private bool _ignoreKHRMaterialsUnlit = false;
+
+        /// <summary>
+        /// Whether to ignore the KHR_materials_unlit extension on the glTF tiles in
+        /// this tileset, if it exists
+        /// </summary>
+        /// <remarks>
+        /// Whether to ignore the KHR_materials_unlit extension on the glTF tiles in
+        /// this tileset, if it exists, and instead render with standard lighting and
+        /// shadows. This property will have no effect if the tileset does not have any
+        /// tiles that use this extension.
+        /// 
+        /// The KHR_materials_unlit extension is often applied to photogrammetry
+        /// tilesets because lighting and shadows are already baked into their
+        /// textures.
+        /// </remarks>
+        public bool ignoreKHRMaterialsUnlit
```

Comment:
This is more consistent with our practice, and also consistent with Microsoft's .NET Framework Design Guidelines. It's `Xml` not `XML`

```suggestion
        public bool ignoreKhrMaterialsUnlit
```